ci-operator/step-registry/opendatahub-io/ci/image-mirror/opendatahub-io-ci-image-mirror-commands.sh

```diff
@@ -75,6 +75,33 @@ if [[ "$IMAGE_TAG" == "YearIndex" ]]; then
     esac
 fi
 
+# Get IMAGE_TAG if it's equal to YearIndex in YYYYq_YYYYMMDD format
+if [[ "$IMAGE_TAG" == "YearIndex" ]]; then
+    YEAR_INDEX=$(echo "$(date +%Y)""$(echo "$(date +%q)"|sed 's/1/a/g;s/2/b/g;s/3/c/g;s/4/d/g')"_"$(date +%Y%m%d)")
+    case "$JOB_TYPE" in
+        presubmit)
+            log "INFO Building YearIndex image tag for a $JOB_TYPE job"
+            IMAGE_TAG="pr-${PULL_NUMBER}-${YEAR_INDEX}"
+            if [[ -n "${RELEASE_VERSION-}" ]]; then
+                IMAGE_TAG="${RELEASE_VERSION}-${IMAGE_TAG}"
+            fi
+            ;;
+        postsubmit)
+            log "INFO Building YearIndex image tag for a $JOB_TYPE job"
+            IMAGE_TAG="${RELEASE_VERSION}-${YEAR_INDEX}"
+            IMAGE_FLOATING_TAG="${RELEASE_VERSION}"
+            ;;
+        periodic)
+            log "INFO Building YearIndex image tag for a $JOB_TYPE job"
+            IMAGE_TAG="${RELEASE_VERSION}-nightly-${YEAR_INDEX}"
+            ;;
+        *)
+            log "ERROR Cannot publish an image from a $JOB_TYPE job"
+            exit 1
+            ;;
+    esac
+fi
+
```

Comment:
Correct me if I am wrong but since this If block of code is a duplicate of the code on [Line 52-76](https://github.com/openshift/release/blob/master/ci-operator/step-registry/opendatahub-io/ci/image-mirror/opendatahub-io-ci-image-mirror-commands.sh#L52) in the same file, I think you wanted to modify the image-mirror-commands script in [step-registry/red-hat-data-services](https://github.com/openshift/release/blob/master/ci-operator/step-registry/red-hat-data-services/ci/image-mirror/red-hat-data-services-ci-image-mirror-commands.sh) not `step-registry/opendatahub-io`

Reply:
Right, fixed!